quarkus-integration-test-infinispan-embedded tests execution fails in native mode with JDK 11

**Describe the bug**
```
2019-12-17 19:31:24,132 ERROR [io.qua.application] (main) Failed to start application: java.lang.RuntimeException: Error injecting org.infinispan.manager.EmbeddedCacheManager io.quarkus.it.infinispan.embedded.TestServlet.emc
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.create(TestServlet_Bean.zig:280)
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.create(TestServlet_Bean.zig:189)
	at io.quarkus.arc.impl.AbstractSharedContext.createInstanceHandle(AbstractSharedContext.java:80)
	at io.quarkus.arc.impl.ComputingCache$CacheFunction.lambda$apply$0(ComputingCache.java:99)
	at io.quarkus.arc.impl.LazyValue.get(LazyValue.java:26)
	at io.quarkus.arc.impl.ComputingCache.getValue(ComputingCache.java:41)
	at io.quarkus.arc.impl.AbstractSharedContext.get(AbstractSharedContext.java:25)
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.get(TestServlet_Bean.zig:73)
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.get(TestServlet_Bean.zig:218)
	at io.quarkus.it.infinispan.embedded.TestServlet_Observer_onStart_fd71b5e0b207b7d1ef838b94eaeff75e52b8f463.notify(TestServlet_Observer_onStart_fd71b5e0b207b7d1ef838b94eaeff75e52b8f463.zig:40)
	at io.quarkus.arc.impl.EventImpl$Notifier.notify(EventImpl.java:224)
	at io.quarkus.arc.impl.EventImpl.fire(EventImpl.java:65)
	at io.quarkus.arc.runtime.LifecycleEventRunner.fireStartupEvent(LifecycleEventRunner.java:23)
	at io.quarkus.arc.runtime.ArcRecorder.handleLifecycleEvents(ArcRecorder.java:108)
	at io.quarkus.deployment.steps.LifecycleEventsBuildStep$startupEvent56.deploy_0(LifecycleEventsBuildStep$startupEvent56.zig:77)
	at io.quarkus.deployment.steps.LifecycleEventsBuildStep$startupEvent56.deploy(LifecycleEventsBuildStep$startupEvent56.zig:36)
	at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:180)
	at io.quarkus.runtime.Application.start(Application.java:87)
	at io.quarkus.runtime.Application.run(Application.java:210)
	at io.quarkus.runner.GeneratedMain.main(GeneratedMain.zig:41)
Caused by: org.infinispan.manager.EmbeddedCacheManagerStartupException: org.infinispan.commons.CacheConfigurationException: Error starting component org.infinispan.xsite.BackupReceiverRepository
	at org.infinispan.manager.DefaultCacheManager.internalStart(DefaultCacheManager.java:746)
	at org.infinispan.manager.DefaultCacheManager.start(DefaultCacheManager.java:712)
	at org.infinispan.manager.DefaultCacheManager.<init>(DefaultCacheManager.java:390)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer.manager(InfinispanEmbeddedProducer.java:47)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.create(InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.zig:90)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.create(InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.zig:134)
	at io.quarkus.arc.impl.AbstractSharedContext.createInstanceHandle(AbstractSharedContext.java:80)
	at io.quarkus.arc.impl.ComputingCache$CacheFunction.lambda$apply$0(ComputingCache.java:99)
	at io.quarkus.arc.impl.LazyValue.get(LazyValue.java:26)
	at io.quarkus.arc.impl.ComputingCache.getValue(ComputingCache.java:41)
	at io.quarkus.arc.impl.AbstractSharedContext.get(AbstractSharedContext.java:25)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.get(InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.zig:43)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.get(InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.zig:343)
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.create(TestServlet_Bean.zig:263)
	... 19 more
Caused by: org.infinispan.commons.CacheConfigurationException: Error starting component org.infinispan.xsite.BackupReceiverRepository
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:555)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startDependencies(BasicComponentRegistryImpl.java:605)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:569)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startDependencies(BasicComponentRegistryImpl.java:605)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:569)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startDependencies(BasicComponentRegistryImpl.java:605)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:569)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startDependencies(BasicComponentRegistryImpl.java:605)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:569)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.AbstractComponentRegistry.internalStart(AbstractComponentRegistry.java:344)
	at org.infinispan.factories.AbstractComponentRegistry.start(AbstractComponentRegistry.java:240)
	at org.infinispan.manager.DefaultCacheManager.internalStart(DefaultCacheManager.java:741)
	... 32 more
Caused by: java.util.MissingResourceException: Resource bundle not found sun.security.utException in thread "main" il.Resources. Register the resource bundle using the option -H:IncludeResourceBundles=sun.security.ujava.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:210)
	at io.quarkus.runtime.Application.start(Application.java:87)
	at io.quarkus.til.Resources.
	at com.oracle.svm.core.jdk.LocalizationSupport.getCached(LocalizationSupport.java:66)
	at java.util.ResourceBundle.getBundle(ResourceBundle.java:53)
	at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1705)
	at sun.security.util.ResourcesMgr.getBundle(ResourcesMgr.java:54)
	at sun.security.util.ResourcesMgr.getString(ResourcesMgr.java:40)
	at javax.security.auth.Subject.getSubject(Subject.java:292)
	at org.infinispan.security.Security.getSubject(Security.java:147)
	at org.infinispan.notifications.impl.AbstractListenerImpl.validateAndAddListenerInvocations(AbstractListenerImpl.java:215)
	at org.infinispan.notifications.cachemanagerlistener.CacheManagerNotifierImpl.addListenerAsync(CacheManagerNotifierImpl.java:172)
	at org.infinispan.notificatioruntime.Application.run(Application.java:210)
	at io.quarkus.runner.GeneratedMain.main(GeneratedMain.zig:41)
Caused by: java.lang.RuntimeException: Error injecting org.infinispan.manager.EmbeddedCacheManager io.quarkus.it.infinispan.embedded.TestServlet.emc
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.create(TestServlet_Bean.zig:280)
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.create(TestServlet_Bean.zig:189)
	at io.quarkus.arc.impl.AbstractSharedContext.createInstanceHandle(AbstractSharedContext.java:80)
	at io.quarkus.arc.impl.ComputingCache$CacheFunction.lambda$apply$0(ComputingCache.java:99)
	at io.quarkus.arc.impl.LazyValue.get(LazyValue.java:26)
	at io.quarkus.arc.impl.ComputingCache.getValue(ComputingCache.java:41)
	at io.quarkus.arc.impl.AbstractSharedContext.get(AbstractSharedContext.java:25)
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.get(TestSerns.Listenable.addListener(Listenable.java:26)
	at org.infinispan.xsite.BackupReceiverRepositoryImpl.start(BackupReceiverRepositoryImpl.java:41)
	at org.infinispan.xsite.CorePackageImpl$1.start(CorePackageImpl.java:39)
	at org.infinispan.xsite.CorePackageImpl$1.start(CorePackageImpl.java:31)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.invokeStart(BasicComponentRegistryImpl.java:587vlet_Bean.zig:73)
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.get(TestServlet_Bean.zig:218)
	at io.quarkus.it.infinispan.embedded.TestServlet_Observer_onStart_fd71b5e0b207b7d1ef838b94eaeff75e52b8f463.notify(TestServlet_Observer_onStart_fd71b5e0b207b7d1ef838b94eaeff75e52b8f463.zig:40)
	at io.quarkus.arc.impl.EventImpl$Notifier.notify(EventImpl.java:224)
	at io.quarkus.arc.impl.EventImpl.fire(EventImpl.java:65)
	at io.quarkus.arc.runtime.LifecycleEventRunner.fireStartupEvent(LifecycleEventRunner.java:23)
	at io.quarkus.arc.runtime.ArcRecorder.handleLifecycleEvents(ArcRecorder.java:108)
	at io.quarkus.deployment.steps.LifecycleEventsBuildStep$startupEvent56.deploy_0(LifecycleEventsBuildStep$startupEvent56.zig:77)
	at io.quarkus.deployment.steps.LifecycleEventsBuildStep$startupEvent56.deploy(LifecycleEventsBuildStep$startupEvent56.zig:36)
	at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:180)
	... 3 more
Caused by: org.infinispan.manager.EmbeddedCacheManagerStartupException: org.infinispan.commons.CacheConfigurationException: Error starting component org.infinispan.xsite.BackupReceiverRepository
	at org.infinispan.manager.DefaultCacheManager.internalStart(DefaultCacheManager.java:746)
	at org.infinispan.manager.DefaultCacheManager.start(DefaultCacheManager.java:712)
	at org.infinispan.manager.DefaultCacheManager.<init>(DefaultCacheManager.java:390)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer.manager(InfinispanEmbeddedProducer.java:47)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.create(InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.zig:90)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.create(InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.zig:134)
	at io.quarkus.arc.impl.AbstractSharedContext.createInstanceHandle(AbstractSharedContext.ja)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:578)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponva:80)
	at io.quarkus.arc.impl.ComputingCache$CacheFunction.lambda$apply$0(ComputingCache.java:99)
	at io.quarkus.arc.impl.LazyValue.get(LazyValue.java:26)
	at io.quarkus.arc.impl.ComputingCache.getValue(ComputingCache.java:41)
	at io.quarkus.arc.impl.AbstractSharedContext.get(AbstractSharedContext.java:25)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer_ProducerMethod_manageentRegistryImpl.java:547)
	... 57 more

r_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.get(InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.zig:43)
	at io.quarkus.infinispan.embedded.runtime.InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.get(InfinispanEmbeddedProducer_ProducerMethod_manager_ed18b755467e1c5fdc8a90c62127a21080a24841_Bean.zig:343)
	at io.quarkus.it.infinispan.embedded.TestServlet_Bean.create(TestServlet_Bean.zig:263)
	... 19 more
Caused by: org.infinispan.commons.CacheConfigurationException: Error starting component org.infinispan.xsite.BackupReceiverRepository
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:555)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startDependencies(BasicComponentRegistryImpl.java:605)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:569)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startDependencies(BasicComponentRegistryImpl.java:605)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:569)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startDependencies(BasicComponentRegistryImpl.java:605)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:569)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startDependencies(BasicComponentRegistryImpl.java:605)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:569)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.access$700(BasicComponentRegistryImpl.java:30)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl$ComponentWrapper.running(BasicComponentRegistryImpl.java:770)
	at org.infinispan.factories.AbstractComponentRegistry.internalStart(AbstractComponentRegistry.java:344)
	at org.infinispan.factories.AbstractComponentRegistry.start(AbstractComponentRegistry.java:240)
	at org.infinispan.manager.DefaultCacheManager.internalStart(DefaultCacheManager.java:741)
	... 32 more
Caused by: java.util.MissingResourceException: Resource bundle not found sun.security.util.Resources. Register the resource bundle using the option -H:IncludeResourceBundles=sun.security.util.Resources.
	at com.oracle.svm.core.jdk.LocalizationSupport.getCached(LocalizationSupport.java:66)
	at java.util.ResourceBundle.getBundle(ResourceBundle.java:53)
	at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1705)
	at sun.security.util.ResourcesMgr.getBundle(ResourcesMgr.java:54)
	at sun.security.util.ResourcesMgr.getString(ResourcesMgr.java:40)
	at javax.security.auth.Subject.getSubject(Subject.java:292)
	at org.infinispan.security.Security.getSubject(Security.java:147)
	at org.infinispan.notifications.impl.AbstractListenerImpl.validateAndAddListenerInvocations(AbstractListenerImpl.java:215)
	at org.infinispan.notifications.cachemanagerlistener.CacheManagerNotifierImpl.addListenerAsync(CacheManagerNotifierImpl.java:172)
	at org.infinispan.notifications.Listenable.addListener(Listenable.java:26)
	at org.infinispan.xsite.BackupReceiverRepositoryImpl.start(BackupReceiverRepositoryImpl.java:41)
	at org.infinispan.xsite.CorePackageImpl$1.start(CorePackageImpl.java:39)
	at org.infinispan.xsite.CorePackageImpl$1.start(CorePackageImpl.java:31)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.invokeStart(BasicComponentRegistryImpl.java:587)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.doStartWrapper(BasicComponentRegistryImpl.java:578)
	at org.infinispan.factories.impl.BasicComponentRegistryImpl.startWrapper(BasicComponentRegistryImpl.java:547)
	... 57 more
```

**To Reproduce**
Steps to reproduce the behavior from the `master` branch:
1. (Optional) Set the `JAVA_HOME` environment variable to a JDK 11 installation directory (OpenJDK 11 or GraalVM 19.3.0 JDK 11 edition). Add the JDK 11 `bin` directory to the path.
2. (Optional) Run `./mvnw clean install -Dmaven.test.skip=true` to build all the Quarkus sources with the JDK 11.
3. Run `./mvnw clean install --projects :quarkus-integration-test-infinispan-embedded -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.0-java11`.

--- a/core/deployment/src/main/java/io/quarkus/deployment/steps/ResourceBundleStep.java
+++ b/core/deployment/src/main/java/io/quarkus/deployment/steps/ResourceBundleStep.java
@@ -1,0 +1,17 @@
+package io.quarkus.deployment.steps;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.nativeimage.NativeImageResourceBundleBuildItem;
+
+public class ResourceBundleStep {
+
+    @BuildStep
+    public NativeImageResourceBundleBuildItem nativeImageResourceBundle() {
+        /*
+         * The following resource bundle sometimes needs to be included into the native image with JDK 11.
+         * This might no longer be required if GraalVM auto-includes it in a future release.
+         * See https://github.com/oracle/graal/issues/2005 for more details about it.
+         */
+        return new NativeImageResourceBundleBuildItem("sun.security.util.Resources");
+    }
+}

